topics/debian/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: debian-linux, debianlinux
+created_by: Ian Murdock
+display_name: Debian
+github_url: https://github.com/debian
+logo: debian.svg
+related: linux, ubuntu
+released: released: September 15, 1993
```

Comment:
Looks like this is what's confusing CI:

```suggestion
released: September 15, 1993
```

Reply:
As the PR author, @n3rd3x3 will have to commit the code suggestion here (if it looks okay).